lasagne/layers/embedding.py

```diff
@@ -18,7 +18,7 @@ class EmbeddingLayer(Layer):
     Parameters
     ----------
     incoming : a :class:`Layer` instance or a tuple
-        The layer feeding into this layer, or the expected input shape
+        The layer feeding into this layer, or the expected input shape.
```

Comment:
ok. I added it because the other parameter decriptions in embedding layer ended with dot.